ansible/tasks/logging/install_fluentd.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create temporary directory on windows host
+  win_tempfile:
+    state: directory
+  register: tempdir_win
+- name: Fetch ruby devkit exe
```

Comment:
nit: Don't address this in PR. I see that you're downloading the file onto the Windows host, while it's not a problem, you've to download it onto every host where we want the package.